src/js/directives.js

```diff
@@ -275,3 +275,9 @@ myApp.directive('strongPassword', function () {
   };
 });
 
+myApp.component('stellarguardResult', {
```

Comment:
My preference is to use components vs element directives. But if you want me to change to element directive that's fine too.